### DISCUSSION TITLE
Create different titles

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#45403] [Q&A]
I want to have a title like `Moz - SEO Software for Smarter Marketing` for the first page of my site and a title like The `Moz Blog [SEO] - Moz` for the rest of my pages.
How do I do this on the `base.html.twig` page?
On the main page, `Moz` is located at the beginning and at the bottom of the rest of the pages.

**Comment by chyshyya** [comment]
Use something like `{{app.request.attributes.get('_route') == 'homepage'}}` to check if current route is you main page or `{{app.request.uri == '/'}}`. There are many ways to do it.
Simple but not nice way is:
```
{{ app.request.attributes.get('_route') == 'homepage' ? 'Moz - ' : '' %}{% block seo_title %}{{some_seo_title}}{% end_block %}{{ app.request.attributes.get('_route') == 'homepage' ? '' : ' - Moz' %}
```
Macro can be used, or some twig extension. Just experiment which way is better for you.
